### DISCUSSION TITLE
Use of #_PAID

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#418] [Q&A] [answered]
Hi,
In a page I use this shortcode [eme_mybookings rsvp_status=1 template_id=19] and in the template_id19 I have this line:
```
[eme_if3 #ESC_PAID value='No'] <a href="#_PAYMENT_URL">pour payer en ligne cliquez ici</a> [/eme_if3]
```
But it does'nt work, whatever the result the link is not displayed 
The question is: can I use #ESC_PAID between [eme_if] and [/eme_if]?

**Comment by liedekef** [comment, marked as answer]
You forgot "tag=":
`[eme_if3 tag='#ESC_PAID' value='No'] <a href="#_PAYMENT_URL">pour payer en ligne cliquez ici</a> [/eme_if3]`
Even better is to check using IS_PAID:
`[eme_if3 tag='#ESC_IS_PAID' value='0'] <a href="#_PAYMENT_URL">pour payer en ligne cliquez ici</a> [/eme_if3]`

**Comment by f14m07** [comment]
Thanks a lot, it's ok now
